automations/js/src/last_week_tonight.mjs

```diff
@@ -69,16 +76,48 @@ const closedIssuesQ = (repo) =>
 const getItemsHtml = (title, items) => {
   if (!items.length) return []
 
+  // Get a unique list of stack labels
+  const stacks = [
+    ...new Set(
+      items
+        .map((item) =>
+          item.labels
+            .map((label) => label.name)
+            .filter((name) => name.startsWith('🧱 stack'))
+        )
+        .flat()
+    ),
+  ].sort()
+  console.log(stacks)
```

Comment:
It's not hugely important but to create the unsorted, non-unique list of stacks you could use:

```ts
 items
   .flatMap(item => item.labels) // Flatten all labels into a single array
   .map(label => label.name) // Extract the name of each label
   .filter(name => name.startsWith('🧱 stack')) 
```